utils.zsh/functions/universal_build

```diff
@@ -3,6 +3,7 @@ autoload -Uz log_info log_status
 local arch
 for arch (x86_64 arm64) {
   (
+    typeset -g -A target_config=(${(kv)target_config//x86_64;arm64/${arch}})
```

Comment:
In `utils.zsh/functions/universal_config`, this near the end of the list. Should they be aligned for consistency?